ci-operator/step-registry/mirror-images/qe-test-images/mirror-images-qe-test-images-commands.sh

```diff
@@ -107,16 +120,23 @@ openshifttest_auth_password=$(cat "/var/run/vault/mirror-registry/registry_quay_
 openshifttest_registry_auth=`echo -n "${openshifttest_auth_user}:${openshifttest_auth_password}" | base64 -w 0`
 jq --argjson a "{\"${MIRROR_REGISTRY_HOST}\": {\"auth\": \"$registry_cred\"},\"quay.io/openshifttest\": {\"auth\": \"${openshifttest_registry_auth}\"}}" '.auths |= . + $a' "${CLUSTER_PROFILE_DIR}/pull-secret" > "${new_pull_secret}"
 
-# MIRROR IMAGES
-# Creating ICSP for quay.io/openshifttest is in enable-qe-catalogsource-disconnected step
-# Set Node CA for Mirror Registry is in enable-qe-catalogsource-disconnected step
-sed -i "s/MIRROR_REGISTRY_PLACEHOLDER/${MIRROR_REGISTRY_HOST}/g" "/tmp/mirror-images-list.yaml" 
+# For disconnected or otherwise unreachable environments, we want to
+# have steps use an HTTP(S) proxy to reach the API server. This proxy
+# configuration file should export HTTP_PROXY, HTTPS_PROXY, and NO_PROXY
+# environment variables, as well as their lowercase equivalents (note
+# that libcurl doesn't recognize the uppercase variables).
+if test -f "${SHARED_DIR}/proxy-conf.sh"
```

Comment:
Once enabled this block, all the mirror traffic would go though proxy to mirror registry, even mirror registry can be directly connected (especially in some QE's scenario on public clouds). Per my original understanding, "${SHARED_DIR}/proxy-conf.sh" is mainly for reaching cluster api endpoints which is not exposed to public.

`mirror-images-qe-test-images` step is bit different with `mirror-images-by-oc-adm`, `mirror-images-qe-test-images` step generally is executed in post-installation, while `mirror-images-by-oc-adm` is executed in pre-installation. 

In  pre-installation phase, when `mirror-images-by-oc-adm` is being executed, `${SHARED_DIR}/proxy-conf.sh` is not created yet in most of cases, so it seldom brings side effect for QE's ci jobs, while in post-installation phase, `${SHARED_DIR}/proxy-conf.sh` is created, so all the traffic will go through proxy to mirror registry. 

To avoid side effect as much as possible, I am thinking the following solutions for your reference:
1. Introduce a separate env to decide source `"${SHARED_DIR}/proxy-conf.sh"` or not, we can set it on job config level as our demand.
2. Create a new proxy config file with a different name, e.g: "${SHARED_DIR}/mirror-proxy-conf.sh" to indicated the file is only used to reach mirror registry behind a proxy.

Reply:
I like idea number 2. If that's ok for you. I'll push it and let you decide. I'm happy with 1 or 2 and would avoid 3 if possible. New steps are expensive to maintain sometimes and this is probably the last step we need from your team, we got a successful CI run with it.

Reply:
If you like option 2, I am okay with it, too. BTW, do you mind to update `mirror-images-by-oc-adm` to use `"${SHARED_DIR}/mirror-proxy-conf.sh"` instead of `"${SHARED_DIR}/proxy-conf.sh"`

Reply:
done.